lambdas/index.ts

```diff
@@ -0,0 +1,47 @@
+import { S3Handler } from 'aws-lambda'
+import * as unzipper from 'unzipper'
+import * as AWS from 'aws-sdk'
+
+const S3 = new AWS.S3()
+
+const BUCKET_NAME = process.env.BUCKET_NAME || ''
```

Comment:
probably want to crash and log if process.env.BUCKET_NAME is `undefined`

Reply:
agreed! BTW, the BUCKET_NAME var comes from the `environment` passed to the `LambdaNodeJs.NodejsFunction` instance, not from the server. But yes, if we forget to pass it inside the code we should throw